site/source/tutorials/9001_dojo1_dijits/index.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: tutorial
+title: Using Dojo 1 Dijits
+overview: In this tutorial, you will learn how to use Dojo 1 Dijits in a Dojo 2 application.
+---
+
+{% section 'first' %}
+
+# Using Dojo 1 Dijits
+
+## Overview
+
+This tutorial will extend the the [Form widgets](../005_form_widgets/) tutorial.  In this tutorial, we will be taking some of the form widgets and replacing them with Dojo 1 Dijits.
+
+While we would hope that developers would author their code fully in Dojo 2, we recognize that it might be challenging to do a wholesale migration, and that using custom Dijits that have been developed for Dojo 1 might help make it _easier_ to migrate an application, migrating complex Dojo 1 Dijits over a period of time.
+
+## Prerequisites
+
+You can [download](../assets/9001_dojo1_dijits-initial.zip) the demo project to get started.
+
+You also need to be familiar with TypeScript as Dojo 2 uses it extensively. For more information, refer to the [TypeScript and Dojo 2](../comingsoon.html) article.
+
+{% section %}
+
+## Adding dependencies
+
+{% task 'Installing dependencies.' %}
+
+The first step will be to install our dependencies using npm:
+
+{% instruction 'Install the initial dependencies via npm.' %}
+
+```
+$ npm install
+```
+
+{% instruction 'Install additional development dependency.' %}
+
+```
+$ npm install dojo-typings --save-dev
+```
+
+{% instruction 'Install additional runtime dependencies.' %}
+
+```
+$ npm install dojo dijit dojo-themes @dojo/interop --save
+```
+
+The `dojo`, `dijit`, and `dojo-themes` packages are the Dojo 1 dependencies we will be using at run-time.  The `@dojo/interop` package contains several tools that allow easy integration with other projects, like Dojo 1 and Redux.
+
+{% section %}
+
+## Updating project configuration
+
+{% task 'Update the `dojo build` configuration.' %}
+
+Dojo 1 and Dijits at runtime are considered _external_ dependencies for the Dojo 2 build system.  We need to supply information to the `dojo build` CLI command to include our external dependencies as part of our build.  The way we provide additional configuration information to the Dojo 2 CLI is via a `.dojorc` file in the root of our project.  The file is a JSON file, where configuration can be provided to each command.
```

Comment:
The way we provide additional configuration information to the Dojo 2 CLI is via a `.dojorc` file in the root of our project. -> We provide additional configuration information to the Dojo 2 CLI via `.dojorc` in the root of our project.